product_brand/views/product_brand_view.xml

```diff
@@ -56,9 +56,11 @@
                             <field name="partner_id" />
                         </group>
                     </group>
-                    <group name="desc" string="Description">
-                        <field name="description" nolabel="1" />
-                    </group>
+                    <notebook>
+                        <page name="desc" string="Description">
```

Comment:
```suggestion
                        <page name="description" string="Description">
```

I wouldn't abbreviate this, but it's just my opinion. not blocking

Reply:
done